### ISSUE TITLE
 Accessibility - Web - Spanish - "Aspectos básicos de un sistema de cumplimiento social" Page

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#126]
Why is there large graphic in the "Términos clave" section why text as part of the image instead of having individual graphics and text next to them? The ALT text for the graphic does not explain the cotton supply system at all. Unless I'm missing something in the text before or after this is not helpful to those using assistive technologies like screen readers.

I'm assuming the same applies for iOS and Android applications.

Please see the attached screenshot.

![complychain - spanish - bascis of social compliance system page - poor alt text](https://user-images.githubusercontent.com/10423487/44424025-b7d82880-a555-11e8-9c91-55fd05800c4e.png)
